lib/jsonapi_errors_handler/errors/standard_error.rb

```diff
@@ -7,7 +8,7 @@ def initialize(title: nil, detail: nil, status: nil, source: {})
         @title = title || "Something went wrong"
         @detail = detail || "We encountered unexpected error, but our developers had been already notified about it"
         @status = status || 500
-        @source = source.deep_stringify_keys
+        @source = ::KeysStringifier.(source)
```

Comment:
When building a gem, all classes should be namespaced to prevent class name clashes with other projects that use it.

Reply:
Thanks for letting me know this, I move it to `lib` folder, because I think it is a general function. I made the changes as your suggestion.